app/main.py

```diff
@@ -1 +1,39 @@
-# write your code here
+class Animal:
+
+    alive = []
+
+    def __init__(self, name: str, health: int = 100) -> None:
+        self.name = name
+        self.health = health
+        self.hidden = False
+        Animal.alive.append(self)
+
+    def die(self) -> None:
+        if self in Animal.alive:
+            Animal.alive.remove(self)
+
+    def __repr__(self) -> str:
+        return (
+            f"{{Name: {self.name}, "
+            f"Health: {self.health}, "
+            f"Hidden: {self.hidden}}}"
+        )
+
+    def __str__(self, cls) -> str:
+        return str([str(animal) for animal in cls.alive])
```

Comment:
The `__str__` method should not have a `cls` parameter. It should be defined as `def __str__(self) -> str:` and should return a string representation of the instance, not the class.